dataduct/etl/etl_actions.py

```diff
@@ -14,8 +15,10 @@
 logger = logging.getLogger(__name__)
 
 
-URL_TEMPLATE = 'https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest'  # noqa
-
+config = Config()
+REGION = config.etl.get('REGION', None)
+URL_TEMPLATE = 'https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest%s'  # noqa
+URL_TEMPLATE %= 'region=%s' % REGION if REGION is not None else ''
```

Comment:
It's because `region` is a query parameter that's supposed to be sent to the server, but everything after the `#` is treated as a browser only thing and not sent to their server.